test/SIL/Parser/unnamed_struct_identifiers.sil

```diff
@@ -1,4 +1,4 @@
-// RUN: %target-sil-opt -sdk %clang-importer-sdk-path %s
+// RUN: %target-sil-opt -I %clang-importer-sdk-path/usr/include %s
```

Comment:
Now that `%target-sil-opt` includes an `-sdk` flag, `sil-opt` complains that two `-sdk` flags were passed in. No other test passes an `-sdk` to `%target-sil-opt` in an individual test file like this, just this old test, so changed this to an include flag instead.

Reply:
Seems that validation-test/SIL/verify_all_overlays.py has the `-sdk` parameter too. It is part of the "long tests", so it might have been filtered out in your testing. That's the only test failing for me in the last version of the code.

Reply:
I'll take a look: I just passed in `-T`, which runs the full validation suite, but probably not the long tests.